uk.co.compucorp.civicrm.tasksassignments/views/dashboard/documents.html

```diff
@@ -6,9 +6,27 @@ <h3 class="panel-title">View Documents By:</h3>
         <div class="panel-body">
             <div class="{{prefix}}filter-date">
                 <ul class="nav nav-pills nav-stacked">
-                    <li role="presentation"><a href ng-model="filterParams.due" uib-btn-radio="'overdue'" uncheckable>{{label.overdue}} <i class="fa fa-calendar" ng-click="viewInCalendar('month');"></i> <span class="badge badge-danger" ng-bind="(documentListOngoing | filterByDue:'overdue' | filterByUserRole:filterParams.userRole.field:filterParams.userRole.isEqual).length"></span></a></li>
-                    <li role="presentation"><a href ng-model="filterParams.due" uib-btn-radio="'dueToday'" uncheckable>{{label.dueToday}} <i class="fa fa-calendar" ng-click="viewInCalendar('day');"></i> <span class="badge badge-primary" ng-bind="(documentListOngoing | filterByDue:'dueToday' | filterByUserRole:filterParams.userRole.field:filterParams.userRole.isEqual).length"></span></a></li>
-                    <li role="presentation"><a href ng-model="filterParams.due"  uib-btn-radio="'dueThisWeek'" uncheckable>{{label.dueThisWeek}} <i class="fa fa-calendar" ng-click="viewInCalendar('week');"></i> <span class="badge badge-primary" ng-bind="(documentListOngoing | filterByDue:'dueThisWeek' | filterByUserRole:filterParams.userRole.field:filterParams.userRole.isEqual).length"></span></a></li>
+                  <li role="presentation">
+                    <a href ng-model="filterParams.due" uib-btn-radio="'overdue'" uncheckable>
+                      {{label.overdue}}
+                      <i class="fa fa-calendar" ng-click="viewInCalendar('month');"></i>
+                      <span class="badge badge-danger" ng-bind="(documentListOngoing | filterByDue:'overdue' | filterByOwnership:filterParams.ownership).length"></span>
+                    </a>
+                  </li>
+                  <li role="presentation">
+                    <a href ng-model="filterParams.due" uib-btn-radio="'dueToday'" uncheckable>
+                      {{label.dueToday}}
+                      <i class="fa fa-calendar" ng-click="viewInCalendar('day');"></i>
+                      <span class="badge badge-primary" ng-bind="(documentListOngoing | filterByDue:'dueToday' | filterByOwnership:filterParams.ownership).length"></span>
+                    </a>
+                  </li>
+                  <li role="presentation">
+                    <a href ng-model="filterParams.due"  uib-btn-radio="'dueThisWeek'" uncheckable>
```

Comment:
Can we place every attribute for the spans in a new line just like it is done for the below "a" tags which makes it more readable.